docs/ci.md

```diff
@@ -73,3 +73,10 @@ If you need to do more intensive build server work - like updating Xcode on the
 To rotate App Store Connect credentials, log into the build server via the `Jenkins` account and update the files in the account's home directory. 
 
 To rotate GitHub credentials, update them via the Jenkins web interface.
+
+## Xcode cloud
+- We are currently using two Xcode Cloud workflows for our builds
+- They can be triggered on Xcode: on the sidebar, click builds -> Cloud -> Look for the chosen build (Main or Experimental) -> right-click on the selected workflow (I.e., Nightly Build) -> Start build, or on App Store Connect
+
+- **Beta** - We currently use Xcode Cloud to run our nightly Beta build. It runs every day at XX UTC. This one is automated. 
```

Comment:
`XX UTC` - looks like some placeholder text slipped in here :) Otherwise looks good!